Embed live W&B Reports (regular reports, not Fully Connected articles) in docs pages (#2948)
wandb/docs: models/reports/embed-reports.mdx

In the upper right of a report, select the **Share** button to open a dialog, then select **Copy embed code**. The copied snippet is an `iframe` HTML element that you can paste into any page that accepts custom HTML.

<Note>

> Only reports that anonymous viewers can open render when embedded: reports in *public* projects, or reports shared with a [view-only link](/models/reports/cross-project-reports#view-only-report-links) (the link's access token is part of the URL).

Reviewer: ```suggestion
You can only embed and render reports from public projects or reports that you've shared using [view-only links](/models/reports/cross-project-reports#view-only-report-links) (the link's access token is part of the URL).
```
More active and direct

Author: Applied in ef9d360bc. Thanks, more direct.
